Jenkinsfile

```diff
@@ -147,7 +147,7 @@ node('ubuntu-zion') {
           OsTools.runSafe(this, """
             docker login --username ${env.DOCKERHUB_API_USERNAME} --password ${env.DOCKERHUB_API_PASSWORD}
           """)
-          OsTools.runSafe(this, "docker push ${organization}/${dockerHubRepository}")
+          OsTools.runSafe(this, "docker push --all-tags ${organization}/${dockerHubRepository}")
```

Comment:
thanks for figuring this out!  it already works for docker-nexus3 👍

Reply:
is there a docker shared pipeline somewhere?

Reply:
you mean a place for these common steps of tagging and pushing to live? I've not seen it.

Reply:
i'm wondering where else the same problem is still lurking around...

Reply:
i intend to visit docker-nexus-iq-cli for good measure, but that one seems to have somehow done the right thing in this last release.